Automattic/wp-calypso
# swapping screenshot with generic info.

This replaces the screenshot in the instructions to get the `Transfer Code`.

Steps to test:
* Go to `https://wordpress.com/setup/google-transfer/domains`.
* Click on the `i` icon next to `Transfer Code` to view the steps.
* Click on `Step 2` to expand it and view the screenshot.
* It should have a generic `mydomain.com` example.
